doc/user-manual/language/universe-levels.lagda.rst

```diff
@@ -84,6 +84,47 @@ With this definition, we have, for example:
     Nat x Set : Set₁
     Set × Set : Set₁
 
+Intrinsic level properties
+--------------------------
+
+.. _intrinsic-level-properties:
+
+Levels and their associated operations have some properties
+which are internally and automatically solved by the compiler.
+This means that we can replace some expressions with others,
+without worrying about the expressions
+for their corresponding levels matching exactly.
+
+For example, for some `F : Level -> Set`, `l1 l2 : Level`,
+`G : Set -> Set -> Set`
+if we have
+
+.. code-block:: agda
+
+    x : F (l1 ⊔ l2)
+
+.. code-block:: agda
+
+    y : F (l2 ⊔ l1)
+
+
+we can use
+
+.. code-block:: agda
+
+    G x y
+
+without having to do any additional work.
+
+
+Here's a list of these properties
+* idempotence - `a ⊔ a` is the same as `a ⊔ a`
+* associativity - `(a ⊔ b) ⊔ c` is the same as `a ⊔ (b ⊔ c)`
+* commutativity - `a ⊔ b` is the same as `b ⊔ a`
+* neutrality of `lzero` - `a ⊔ lzero` is the same as `a`
+* subsumption - `a ⊔ lsuc a` is the same as `lsuc a`
```

Comment:
This also holds more generally for an arbitrary number of `lsuc`s - I'm not sure if those can be derived from the laws given here, so it could be worth mentioning explicitly

Reply:
I guess you can actually derive it as follows: `a ⊔ lsuc (lsuc a) == a ⊔ (lsuc a ⊔ lsuc (lsuc a)) == (a ⊔ lsuc a) ⊔ lsuc (lsuc a) == lsuc a ⊔ lsuc (lsuc a) == lsuc (lsuc a)`. Still, mentioning it doesn't hurt.

Reply:
I wrote a bit about this